### PR TITLE
Rhui 3 updates

### PR DESCRIPTION
**Pull request by cantlep**
Please review additional notes in RHUI 2 documentation prompting reader to check RHUI 3 documentation

### CODE REVIEW

File: articles/shared/shared-how-install-rhui-vm-3.md [hunk 1]
```diff
@@ -53,6 +53,9 @@ There are two prerequisites for performing this operation:
 
 ### Locations
 
+> [!NOTE]
+> These locations are only available from within the UKCloud Network
```
Comment (suehighmoor) [inline articles/shared/shared-how-install-rhui-vm-3.md]: Would "accessible" work better than "available"

Comment (cantlep) [reply articles/shared/shared-how-install-rhui-vm-3.md]: Yes :-)

**Review by suehighmoor — APPROVED** [review]
LGTM

**Review by mikepukcloud — APPROVED** [review]
LGTM

**Pull request merged**